### ISSUE TITLE
CE-125 battery replacement option?

### ISSUE THREAD

**Issue by genivia-inc** [number42net/sharp-pc1251#1]
I own a CE-125 with PC-1262. Both work well. But the CE-125 batteries died long ago and I want to remove them.

Even better, it appears that replacement batteries may be available according to this blog (use Google translate to read):

http://kyoro205.blog.fc2.com/blog-entry-65.html

Here are the batteries referenced:

https://www.amazon.co.jp/dp/B002DI4G20/ref=as_sl_pc_qf_sp_asin_til?tag=takamame0205-22&linkCode=w00&linkId=&creativeASIN=B002DI4G20

I have not tried the replacements. I am curious if these might work?

**Comment by number42net** [comment]
Those would probably work, but are just single cells and unless you're in Japan, shipping will be expensive. Luckily the CE-125 uses a very common flat pack of 4 Ni-Cad 1.2v AA batteries, so pretty much any replacement will work, usually they're called 4.8v Ni-Cad batteries. I see you're located in the US, so here is an example from [Amazon.com](https://www.amazon.com/Qsmily-Rechargeable-Connector-Suitable-Excavator/dp/B08LL38PDZ/ref=sr_1_23?dchild=1&keywords=Ni-Cad+4.8v&qid=1612172296&sr=8-23)

There are pictures of the original pack and how it's soldered into the board on my [restore blog](https://retro.engineer/projects/2021-01-16_Sharp_PC-1251/) as well. Unless the batteries leaked and the corrosion made it to the board, it should be a very quick swap with a soldering ion. (or even without, but you'll have to splice the cables of the new battery pack to the old ones)

**Comment by genivia-inc** [comment]
I ordered the sealed 4.8V battery pack. Thanks for the tip!

In my case the SHARP battery leaked more than yours. The battery looked awful. Luckily it had not sprinkled the deposits throughout the case, so cleaning up the case wasn't too hard. There is some buildup of deposits on the exposed solder points on the PCB where the power leads are connected. I removed it with a bit of distilled white vinegar and cleaned it up with swabs with a bit of alcohol. The PCB does not appear to be affected I think, because I was able to measure continuity. Time will tell, once I get the batteries. The deposits look the same as in Kyoro's blog http://blog-imgs-49.fc2.com/k/y/o/kyoro205/CE125.jpg visible on the bottom left of the PCB close to the battery.

Perhaps I have a later CE-125 model than yours, because there is no orange connector where the test opening is located in the back, as is also shown in Kyoro's blog. They probably decided to leave it out to save cost.

The cassette player works and runs smoothly. It may fail at some point, I suppose I will fix it then. I don't use it often, but it is nice and nostalgic that it still works. I mostly use the bin2img and bin2wav tools to `cload` programs on my PC-1262. The PC-1262 is one of my favorite vintage calculators. For the time back then it is quite powerful. With some calc/math code is loaded it can still compete with modern non-graphing calculators, while offering programmability.

While I was working on the CE-125, I also inspected my PC-2500 machine that I recently acquired from eBay. The battery leakage was also bad, but this machine has more space inside so there was no deposit buildup anywhere fortunately. The sealed battery pack is a 6V with 5 AA batteries arranged as 2+3 to fit the box. I ordered 5 NiCd AA batteries with solder leads to reproduce the 2+3 shape and use the original JST connector with wires. Not sure how to seal the battery though, if this is recommended?
